src/components/modals/flow-modal/README.md

```diff
@@ -0,0 +1,58 @@
+# FlowModal
+
+The flow modal dialog.
+
+## Usage
+
+The `<FlowModal>` component should be used along with `<FlowModalStep>`.
+
+```jsx
+import { FlowModal } from '@nomios/web-uikit';
+
+<ModalFlow open variant="simple" step="first">
```

Comment:
This should be: 

```jsx
<FlowModal open variant="simple" step="first">
```

instead of:

```jsx
<ModalFlow open variant="simple" step="first">
```